android/src/main/kotlin/com/bitmovin/player/flutter/Util.kt

```diff
@@ -25,3 +28,10 @@ inline fun postToMainThread(crossinline block: () -> Unit?) {
         block()
     }
 }
+
+fun Context.requireActivity(): Activity =
+    when (this) {
+        is Activity -> this
+        is ContextWrapper -> baseContext.requireActivity()
+        else -> error("No activity found for context $this")
+    }
```

Comment:
NIT: I am not a big fan of a single big util file. As it is - for now - only used once, can we just have it private in the  `FlutterPlayerView.kt`?  If you already have another use case for this in mind, please create a more specific file like e.g. `ContextExtension.kt`.

Reply:
Maybe we can also have a slightly more verbose error message in case we don't have an activity. 🙂

Reply:
You're right big utils class aren't pretty. Not sure yet if we'll need this again so I made it private.
[d315606](https://github.com/bitmovin/bitmovin-player-flutter/pull/85/commits/d315606fa0c0060c3576aee1f714913a588f7a65)